adapter/internal/eventhub/subscription.go

```diff
@@ -116,6 +119,32 @@ func LoadSubscriptionData(configFile *config.Config) {
 	var responseChannel = make(chan response)
 	for _, url := range resources {
 		go InvokeService(url.endpoint, url.responseType, nil, responseChannel, 0)
+		for {
+			data := <-responseChannel
+			logger.LoggerSync.Debug("Receiving data for an environment")
```

Comment:
Please make this debug log more informative

Reply:
done